discovery_engine_core/web-api/openapi.yaml

```diff
@@ -127,6 +127,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/IngestionError'
+    delete:
+      tags:
+        - documents
+      summary: Deletes all listed documents.
+      description: |-
+        Deletes all documents listed in the request body. The endpoint is
+        idempotent. I.e. If the list contains one or multiple non-existing
+        document it does not produce  an error.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DeleteDocumentsRequest'
+      responses:
+        '204':
+          description: successful operation
+        '400':
+          description: invalid request
+
+  /documents/{document_id}:
```

Comment:
❔ Only the endpoint above is mentioned in the ticket. was it later decided that we also need this endpoint?

Reply:
historically we once only had `DELETE /documents/{document_id}` and then moved to batched delete as main functionality it's not clear if only having batched delete was intentional or accidental but as long as single delete is time wise cheap to implement it would be nice to have 

daily: we can have it

me: We might just implement it by using bulk/batch request with a single element.